mllib/src/main/scala/org/apache/spark/mllib/optimization/NNLS.scala

```diff
@@ -53,8 +53,11 @@ private[spark] object NNLS {
    * projected gradient method.  That is, find x minimising ||Ax - b||_2 given A^T A and A^T b.
    *
    * We solve the problem
+   *
+   * {{{
    *   min_x      1/2 x^T ata x^T - x^T atb
-   *   subject to x &gt;= 0
+   *   subject to x >= 0
+   * }}}
```

Comment:
I think this was meant to render as LaTeX with `$$`; does that work? Otherwise, leave it but change the last bit to "where x is nonnegative"